courseraprogramming/commands/config.py

```diff
@@ -59,7 +59,7 @@ def check_auth(args):
             r.text)
         name = None
 
-    if not args.quiet > 0:
```

Comment:
This will only work in Python 2 because in Python 2 you can compare `None` with 0 but not in Python 3